src/rgw/rgw_sal_motr.cc

```diff
@@ -1932,9 +1937,16 @@ int MotrObject::write_mobj(const DoutPrefixProvider *dpp, bufferlist&& in_buffer
     if (left < bs)
       bs = this->get_optimal_bs(left);
     if (left < bs) {
+      // This is the last I/O.
```

Comment:
Do we need any changes during the read as well?

Reply:
its done in one of previous code changes in MotrObject::read_mobj

Reply:
For reading, we used int64_t data type. No other change is required in reading